ouroboros-consensus-cardano-test/test/Test/ThreadNet/MaryAlonzo.hs

```diff
@@ -0,0 +1,384 @@
+{-# LANGUAGE DeriveGeneric        #-}
```

Comment:
This whole module is a copy-paste. Again, we're kicking the can down the road: I'll be returning shortly to the ThreadNet rewrite work, and that will fully supplant this module and all those like it.